### ISSUE TITLE
Modularisation of the whole project.

### ISSUE THREAD

**Issue by garimasingh128** [garimasingh128/profext#2]
Put all JS Files and images in a directory and update the README as per the changes.

**Comment by anmolkaur18** [comment]
I am interested 🙋🏻‍♀️ please assigning to me @garimasingh128

**Comment by garimasingh128** [comment]
@anmolkaur18 You can start working on it now!

**Comment by sauravvsonu** [comment]
can I work on this, maintain a folder structure, and update the readME

**Comment by sauravvsonu** [comment]
@garimasingh128, hey, I think images and Js file have been moved into a folder in this issue. So, could you help me with what to do? Next.

**Comment by garimasingh128** [comment]
yup I did it myself as some participants were getting confused about what each file has to do. So I will make this as stale as of now and we can see if some modularisation is required by the end of the program.
